resolwe_bio/processes/plots/bamplot.yml

```diff
@@ -0,0 +1,222 @@
+# ========
+# Bamplot
+# ========
+---
+
+- slug: bamplot
+  name: bamplot
+  requirements:
+    expression-engine: jinja
+  data_name: "{{ alignments.bam.file|basename|default('?') }}"
+  version: 0.0.1
+  type: data:bam:plot
+  category: plotting
+  persistence: CACHED
+  description: >
+    Plotting a single locus from a bam.
+  input:
+    - name: g
+      label: Genome
+      type: basic:string
+      choices:
+        - label: HG19
+          value: HG19
+        - label: HG18
+          value: HG18
+        - label: MM8
+          value: MM8
+        - label: MM9
+          value: MM9
+        - label: MM10
+          value: MM10
+    - name: i_gff
+      label: Region string
+      type: data:annotation
+      required: false
+      description: >
+         Enter .gff file.
+    - name: i_region
+      label: Region string
+      type: basic:string
+      required: false
+      description: >
+         Enter genomic region e.g. chr1:+:1-1000.
+    - name: b
```

Comment:
Use more verbose attribute names. It is much easier to understand code with `bam`, `color`, `sense`..., than just `b`, `c`, `s`...